base_classes/NXcg_half_edge_data_structure.nxdl.xml

```diff
@@ -54,41 +54,68 @@
          Such a data structure can be used to efficiently circulate around faces
```

Comment:
Also say it is known as doubly connected edge list

Reply:
Implemented